website/static/docs/external_plugins/examples/sshfs.sh

```diff
@@ -145,7 +157,10 @@ function print_children {
   # The code here is equivalent to ls'ing the directory, then invoking
   # `test-d` and stat on each entry to obtain the following information:
   #   <is_dir> <sizeAttr> <atime> <mtime> <ctime> <mode> <path>
-  stat_output=`vm_exec ${vm} "find ${dir} -mindepth 1 -maxdepth 1 | xargs -n1 -r -I {} bash -c '(test -d \\$@; echo -n \"\\$? \") && stat -c \"%s %X %Y %Z %f %n\" \\$@' _ {}"`
+  #
+  # TODO: Handle TTY strangeness in the returned output. For now, it is
```

Comment:
This was causing missing chunks of output from the printed JSON. I.e. things looked something like:
```
[
","methods":["list"],"attributes":{"atime":1558322711,"mtime":1558133886,"ctime":1558133886,"mode":16749,"size":4096}},
","methods":["list"],"attributes":{"atime":1558133854,"mtime":1558133854,"ctime":1558133854,"mode":16749,"size":0}},
","methods":["list"],"attributes":{"atime":1558322715,"mtime":1558133858,"ctime":1558133858,"mode":16877,"size":3120}},
","methods":["list"],"attributes":{"atime":1558246965,"mtime":1558133869,"ctime":1558133869,"mode":16877,"size":840}},
","methods":["list"],"attributes":{"atime":1558245893,"mtime":1558133854,"ctime":1558133854,"mode":16749,"size":0}},
","methods":["list"],"attributes":{"atime":1558330785,"mtime":1558136154,"ctime":1558136154,"mode":16744,"size":4096}},
","methods":["list"],"attributes":{"atime":1558323061,"mtime":1558133868,"ctime":1558133868,"mode":16877,"size":8192}},
","methods":["list"],"attributes":{"atime":1558133859,"mtime":1558323061,"ctime":1558323061,"mode":17407,"size":4096}},
","methods":["list"],"attributes":{"atime":1558311781,"mtime":1558133856,"ctime":1558133858,"mode":16877,"size":4096}},
","methods":["list"],"attributes":{"atime":1558330786,"mtime":1547678697,"ctime":1547678697,"mode":16877,"size":4096}},
","methods":["list"],"attributes":{"atime":1558306860,"mtime":1547678697,"ctime":1547678697,"mode":57855,"size":7}},
","methods":["list"],"attributes":{"atime":1558306861,"mtime":1547678697,"ctime":1547678697,"mode":57855,"size":8}},
","methods":["list"],"attributes":{"atime":1558330784,"mtime":1547678697,"ctime":1547678697,"mode":57855,"size":7}},
","methods":["list"],"attributes":{"atime":1558306668,"mtime":1547678697,"ctime":1547678697,"mode":57855,"size":9}},
","methods":["list"],"attributes":{"atime":1432543056,"mtime":1432543056,"ctime":1547678697,"mode":16877,"size":6}},
","methods":["list"],"attributes":{"atime":1558246933,"mtime":1432543056,"ctime":1547678697,"mode":16877,"size":6}},
","methods":["list"],"attributes":{"atime":1558322709,"mtime":1547679200,"ctime":1547679200,"mode":16877,"size":17}},
","methods":["list"],"attributes":{"atime":1558322710,"mtime":1547679439,"ctime":1547679439,"mode":16877,"size":6}},
","methods":["list"],"attributes":{"atime":1558322710,"mtime":1432543056,"ctime":1547678697,"mode":16877,"size":6}},
","methods":["read","stream"],"attributes":{"atime":1547678985,"mtime":1547678985,"ctime":1547678985,"mode":33188,"size":0}}
]
```

when they should instead look something like
```
[
{"name":"dev","methods":["list"],"attributes":{"atime":1558322715,"mtime":1558133858,"ctime":1558133858,"mode":16877,"size":3120}},
{"name":"boot","methods":["list"],"attributes":{"atime":1558322711,"mtime":1558133886,"ctime":1558133886,"mode":16749,"size":4096}},
{"name":"proc","methods":["list"],"attributes":{"atime":1558133854,"mtime":1558133854,"ctime":1558133854,"mode":16749,"size":0}},
{"name":"run","methods":["list"],"attributes":{"atime":1558246965,"mtime":1558133869,"ctime":1558133869,"mode":16877,"size":840}},
{"name":"sys","methods":["list"],"attributes":{"atime":1558245893,"mtime":1558133854,"ctime":1558133854,"mode":16749,"size":0}},
{"name":"root","methods":["list"],"attributes":{"atime":1558330785,"mtime":1558136154,"ctime":1558136154,"mode":16744,"size":4096}},
{"name":"etc","methods":["list"],"attributes":{"atime":1558323061,"mtime":1558133868,"ctime":1558133868,"mode":16877,"size":8192}},
{"name":"tmp","methods":["list"],"attributes":{"atime":1558133859,"mtime":1558323061,"ctime":1558323061,"mode":17407,"size":4096}},
{"name":"var","methods":["list"],"attributes":{"atime":1558311781,"mtime":1558133856,"ctime":1558133858,"mode":16877,"size":4096}},
{"name":"usr","methods":["list"],"attributes":{"atime":1558330786,"mtime":1547678697,"ctime":1547678697,"mode":16877,"size":4096}},
{"name":"bin","methods":["list"],"attributes":{"atime":1558306860,"mtime":1547678697,"ctime":1547678697,"mode":57855,"size":7}},
{"name":"sbin","methods":["list"],"attributes":{"atime":1558306861,"mtime":1547678697,"ctime":1547678697,"mode":57855,"size":8}},
{"name":"lib","methods":["list"],"attributes":{"atime":1558330784,"mtime":1547678697,"ctime":1547678697,"mode":57855,"size":7}},
{"name":"lib64","methods":["list"],"attributes":{"atime":1558306668,"mtime":1547678697,"ctime":1547678697,"mode":57855,"size":9}},
{"name":"home","methods":["list"],"attributes":{"atime":1432543056,"mtime":1432543056,"ctime":1547678697,"mode":16877,"size":6}},
{"name":"mnt","methods":["list"],"attributes":{"atime":1558322709,"mtime":1547679200,"ctime":1547679200,"mode":16877,"size":17}},
{"name":"media","methods":["list"],"attributes":{"atime":1558246933,"mtime":1432543056,"ctime":1547678697,"mode":16877,"size":6}},
{"name":"opt","methods":["list"],"attributes":{"atime":1558322710,"mtime":1547679439,"ctime":1547679439,"mode":16877,"size":6}},
{"name":"srv","methods":["list"],"attributes":{"atime":1558322710,"mtime":1432543056,"ctime":1547678697,"mode":16877,"size":6}},
{"name":".autorelabel","methods":["read","stream"],"attributes":{"atime":1547678985,"mtime":1547678985,"ctime":1547678985,"mode":33188,"size":0}}
]
```